dts/bindings/flash_controller/flash-controller.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
How come we didn't have this before, if flash controllers were already being used in the `.dtsi` files?

Reply:
Not flash controller was used - but its sub-node for flash memory description.

Reply:
Ok, understood. This looks good to me then.